src/libraries/System.Reflection.MetadataLoadContext/tests/src/SampleMetadata/SampleMetadata.cs

```diff
@@ -198,6 +198,8 @@ public class ParametersWithDefaultValues
         public void Foo4(short s = -34) { }
         public void Foo5(decimal d = 1234m) { }
         public void Foo6([DateTimeConstant(ticks: 8736726782)] DateTime dt) { }
+        public void Foo7(string s1 = "foo", string s2 = "") { }
```

Comment:
NIT: maybe add a test case for null default?
```suggestion
        public void Foo7(string s1 = "foo", string s2 = "", string s3 = null) { }
```

Reply:
Sure, done.